textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/keywordmatch/KeywordMatcher.java

```diff
@@ -0,0 +1,168 @@
+package edu.uci.ics.textdb.dataflow.keywordmatch;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.apache.lucene.search.Query;
```

Comment:
org.apache.lucene.search.Query object is used in line 33 on KeywordPredicate.java